packages/@okta/vuepress-site/docs/guides/token-inline-hook/main/index.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Token Inline Hook
+excerpt: Learn how to easily implement a Token Inline hook
+layout: Guides
+---
+
+<StackSelector />
+
+This guide provides a working example of an Okta Token Inline Hook. It uses the website [Glitch.com](https://glitch.com) to act as an external service and receive and respond to Token Inline Hook calls.
+
+---
+
+**Learning outcomes**
+
+* Understand the Okta Token Inline Hook calls and responses.
+* Implement a simple working example of a Token Inline Hook with a Glitch.com project, which acts as an external service.
+* Test the Token Inline hook.
```

Comment:
```suggestion
* Test the Token Inline Hook.
```
"Hook" was previously capitalized, so I'm following the convention.

Reply:
Good catch. Thanks.